### DISCUSSION TITLE
Why is breitbart.com labeled as fakenews?

### DISCUSSION THREAD

**Discussion by rodrigopolo** [StevenBlack/hosts#1713] [Q&A]
After watching a Linus Tech Tips video I applied this list to my system, then I found out that breitbart.com was included as fakenews, Why? CNN and MSNBC and all that pushed the Russian collusion hoax should also be included?

**Comment by StevenBlack** [comment]
@rodrigopolo  here is a quote from the first paragraph of the [Breitbart News English Wikipedia article](https://en.wikipedia.org/wiki/Breitbart_News). The article is based on 210 citations.

> Breitbart News Network (known commonly as Breitbart News, Breitbart or Breitbart.com) is an American far-right syndicated news, opinion, and commentary website founded in mid-2007 by American conservative commentator Andrew Breitbart, who conceived it as "the Huffington Post of the right". Its journalists are widely considered to be ideologically driven, and much of its content has been called misogynistic, xenophobic, and racist by liberals and traditional conservatives alike. The site has published a number of conspiracy theories, instances of anti-Chinese xenophobia, and intentionally misleading stories.

I'm sorry, but you appear to be truly basic. Your epistemology is problematic. I recommend you consider addressing that.

cc @marktron

**Reply by rodrigopolo** [reply]
>"Larry Sanger, who is the co-founder of Wikipedia, is ringing alarm bells warning that the information source has been taken over by leftists who are eradicating any news that doesn’t fit their political agenda and that it can no longer be trusted."
Source (video): https://www.air.tv/watch?v=mWwmiBtPRnWtNoeVh0NLlw

In other words, Wikipedia could say whatever the editors want to say, but, it has any facts that base any claim? And again, CNN and other news outlets pushed the Russian collusion hoax, and are not listed in the fake news list.

BTW, English is not my native language, nevertheless, this is not about epistemology, it is about fact based lists, and it's about treating everyone the same based on facts, not in opinions that can be tinted by ideology and third parties agendas. 

Edit: Maybe a left and right leaning lists could fix the problem ¯\_(ツ)_/¯

**Reply by StevenBlack** [reply]
@rodrigopolo thank you for that quote from air.tv.

If you follow the link to the [Breitbart News English Wikipedia article](https://en.wikipedia.org/wiki/Breitbart_News) I posted for you, take note that the article is extensively backed by citations.

Is fake news a judgment call?  Absolutely.  However one has to draw the line somewhere.  I personally agree with @marktron, who is the arbiter here.  In my view, it's not even a close-call in this instance.

I'm gonna lock this thread now because I have a limited capacity for bullshit.
